watcher_namespaces.go

```diff
@@ -210,6 +208,20 @@ func (nw *NamespaceWatcher) refreshNamespaceStats(singleInfoKey string, infoKeys
 			// push to prom-channel
 			pushToPrometheus(asMetric, pv, labels, labelValues, ch)
 		}
+
+		// below code section is to ensure ns+latencies combination is handled during LatencyWatcher
+		//
+		// check and if latency benchmarks stat - is it enabled (bool true==1 and false==0 after conversion)
+		if isStatLatencyHistRelated(stat) {
+			delete(LatencyBenchmarks, nsName+"-"+stat)
+
+			if pv == 1 {
+				LatencyBenchmarks[nsName+"-"+stat] = stat
+			}
+		}
+		// append default re-repl, as this auto-enabled, but not coming as part of latencies, we need this as namespace is available only here
+		LatencyBenchmarks[nsName+"-latency-hist-re-repl"] = "{" + nsName + "}-re-repl"
```

Comment:
This should be done outside the for loop. isnt it ?
Else, it will get added again and again to the map.
Luckily this is not a bug as the same element gets added to the map.